src/GoogleSheets/packages/auth/verify/confirm/main.go

```diff
@@ -0,0 +1,57 @@
+package VerifyEmployeeCode
+
+import (
+	"GoogleSheets/packages/common/Constants/AuthConstants"
+	"GoogleSheets/packages/common/Constants/SharedConstants"
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+)
+
+func HandleRequest(ctx context.Context, event events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	var confirmReq AuthConstants.ConfirmCodeRequest
+	err := json.Unmarshal([]byte(event.Body), &confirmReq)
+	if err != nil {
+		return events.APIGatewayProxyResponse{
+			StatusCode: 400,
+			Headers:    SharedConstants.ALLOW_ORIGINS_HEADER,
+			Body:       "Invalid request body",
+		}, nil
+	}
+
+	sess := session.Must(session.NewSession())
+	cognitoClient := cognitoidentityprovider.New(sess)
+
+	confirmInput := &cognitoidentityprovider.ConfirmSignUpInput{
+		ClientId:         aws.String(os.Getenv("COGNITO_CLIENT_ID")),
+		Username:         aws.String(confirmReq.Email),
+		ConfirmationCode: aws.String(confirmReq.Code),
+	}
+
+	_, err = cognitoClient.ConfirmSignUp(confirmInput)
+	if err != nil {
```

Comment:
`err` can be many different errors, and we want to decipher whether its an actual system error (like the lambda failed) or if it's a user error like a code mismatch. If it's a user error, we want to return `400` status with a useful error message so it can be displayed on the frontend alert.

 Please look through the SDK docs and figure out ones we should look for.
See [this as an example](https://github.com/jlui17/W2FHR/pull/6/files#diff-32e84a38a86b2ccd7198cc3e3865a7db9d3ca099c42be6b888a91c8253ab3459R73-R99).